.travis.yml

```diff
@@ -70,3 +70,17 @@ script:
 - python mat_eqn.py
 - python mdf.py
 - ls
+- cd ..
+
+before_deploy: 
+- pwd
+- echo "$PYPI_TOKEN"
+  
+deploy:
```

Comment:
I'm not familiar with the deploy stuff. Is it triggered on every merge to master?

Reply:
On on tagged commits

Reply:
But we could change that with the 'on` setting in deployment

Reply:
Sounds good.